modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -338,6 +342,26 @@ sub core_pipeline_analyses {
             -rc_name    => '2Gb_job',
         },
 
+        {   -logic_name => 'fasta_dumps_per_collection_factory',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PassFastaDumpsPerCollection',
+            -parameters => {
+                'symlink_dir'          => $self->o('shared_fasta_dir'),
+                'ref_member_dumps_dir' => $self->o('ref_member_dumps_dir'),
+            },
+            -rc_name    => '500Mb_job',
+            -flow_into  => [ 'symlink_fasta_to_shared_loc' ],
+            -wait_for   => [ 'create_reference_sets' ],
```

Comment:
There is indeed a reason - it involves semaphores. My first attempt was to flow from `create_reference_sets`.

Reply:
The issues in lastz were due to the complexity of the pipeline and the mixing in/introductions of semaphores and waiting across multiple analyses. The preference to semaphores is because semaphores create a more dynamic way of running the pipelines, jobs are dependent on individual jobs and not whole analyses etc. In this case the `wait_for` really has to "wait for" the entire analysis, so it is totally appropriate and the correct use. There is little point in removing this because the "race" condition will not occur in this pipeline, furthermore, this is a rapid release specific pipeline, it will not be long lived.